spec/turn_spec.rb

```diff
@@ -2,5 +2,52 @@
 require './lib/card'
 
 RSpec.describe Turn do
+  it 'exists' do
```

Comment:
If you wanted to DRY up your tests, you could use a before action in your test so you don't have to keep typing card = Card.new(...)